File: app/src/main/java/com/example/architechturestartercode/presentation/allmovies/presenter/AllMoviesPresenter.java
package com.example.architechturestartercode.presentation.allmovies.presenter;

import com.example.architechturestartercode.data.movie.model.Movie;

public interface AllMoviesPresenter {
    void getAllMovies();
    void addToFav(Movie movie);

    void openMovieDetails(Movie movie);
}

File: app/src/main/java/com/example/architechturestartercode/presentation/allmovies/presenter/AllMoviesPresenter.kt
package com.example.architechturestartercode.presentation.allmovies.presenter

import com.example.architechturestartercode.data.movie.model.Movie

interface AllMoviesPresenter {
    fun getAllMovies()
    fun addToFav(movie: Movie)
    fun openMovieDetails(movie: Movie)
}

